app/templates/contact.hbs

```diff
@@ -20,6 +20,15 @@
 					showErrorState=showErrorState}}
 			</div>
 		</div>
+    {{#if showErrorState}}
+      <div>
```

Comment:
@mkong0216 This is the template to display error messages. feel free to tweak!

Reply:
okay will do! should I add it to this PR/branch or let you merge it in first?

Reply:
This PR already does lots of things, so let's do this in separate pr!